adhocteam/Head-Start-TTADP
# Cm 60 add nightly reconcilliation

**Description of change**
Adds job to worker to process legacy report reconciliation

**How to test**
uncomment line 165 of `src/services/legacyreports.js` and comment out line 166. That will make the reconciliation run every minute instead of every night at 2am. Start the app and watch the worker logs. You should see it start the reconciliation process every minute.

If you import the legacy reports and change the users email addresses as describted in #255, you should see the legacy reports get updated within a minute of changing the email address.

**Issue(s)**
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-72
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-60

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
